.github/workflows/build.yml

```diff
@@ -23,6 +23,7 @@ jobs:
         go:
           - 1.17
```

Comment:
I wonder if we should remove `1.17` from GHA -- it's the last pre-generics version and might be causing all kinds of unnecessary problems for folks who started using generics 🤔

Reply:
updated